sql/telemetry/client_probe_counts_v1/query.sql

```diff
@@ -81,8 +81,17 @@ RETURNS ARRAY<STRUCT<key STRING, value FLOAT64>> AS (
   )
 );
 
-CREATE TEMP FUNCTION udf_get_buckets(min INT64, max INT64, num INT64, metric_type STRING)
+CREATE TEMP FUNCTION udf_to_string_arr(buckets ARRAY<INT64>)
```

Comment:
So right now the "histogram" for a boolean uses the keys `['always','never','sometimes']` which are strings. In order to keep both boolean and scalar (and anything else) in the same table, I cast all keys to strings.

Any thoughts on an alternative? I guess we could make always/sometimes/never be represented by 0/1/2. Just figured it would be nice for the UI (or any user querying the data) to have these to access directly

Reply:
okay, I see. It is a bit confusing right now but we can think through an alternative at a later stage.